spring-cloud-task-docs/src/main/asciidoc/batch.adoc

```diff
@@ -64,11 +64,16 @@ look like the following:
 @Bean
 public PartitionHandler partitionHandler(TaskLauncher taskLauncher,
 		JobExplorer jobExplorer) throws Exception {
-	MavenResource resource =
+
+	MavenProperties mavenProperties = new MavenProperties();
```

Comment:
actually could just use `MavenResourceLoader` directly... but I guess what's shown now is fine - it's a bit lower level though